### PR TITLE
Initial version of xcdchk-testsuite for Jump

### PR DESCRIPTION
**Pull request by lkocman**
* Resolves Issue #2449

DO NOT MERGE

TODO 

- [x] make sure that it can be executed locally (I'm still getting some missing attr errors on python3).
- [ ] ask Thorsten if it's okay to share his tests on github (currently in gitlab)
- [ ] wrap kukuk/xcdchk
- [ ] handle dependencies on rpmdev-tools, libsolv-tools (installcheck)
- [ ] do not use /data as a data storage for xcdchk

Tool itself was moved here https://github.com/openSUSE/xcdchk

### CODE REVIEW

File: gocd/xcdchk.jump.opensuse.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      script:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+    timer:
+      spec: 0 */10 * ? * *
+      only_on_changes: false
+    stages:
+    - Run:
+        approval: manual
+        resources:
+          - staging-bot
```
Comment (coolo) [inline gocd/xcdchk.jump.opensuse.yaml:18]: staging-bot is for quick jobs. this sounds like an expensive job and as such should be in the repo-checker resource

File: gocd/xcdchk.jump.opensuse.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+format_version: 3
+pipelines:
+  openSUSE.QA.Maint:
```
Comment (coolo) [inline gocd/xcdchk.jump.opensuse.yaml:3]: this pipeline is something else

File: xcdchk-testsuite.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python2
```
Comment (coolo) [inline xcdchk-testsuite.py:1]: there is for sure no python2 in gocd

File: gocd/xcdchk.jump.opensuse.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    group: openSUSE.Checkers
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-xcdchk
+    materials:
+      script:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+    timer:
+      spec: 0 */10 * ? * *
```
Comment (coolo) [inline gocd/xcdchk.jump.opensuse.yaml:12]: it doesn't make sense to have this script run every 10 minutes - you want this as part of the release pipeline, not as a checker

File: xcdchk-testsuite.py (hunk 3 above)
Comment (lkocman) [reply xcdchk-testsuite.py]: fixed

File: gocd/xcdchk.jump.opensuse.yaml (hunk 1 above)
Comment (lkocman) [reply gocd/xcdchk.jump.opensuse.yaml]: Fixed

File: gocd/xcdchk.jump.opensuse.yaml (hunk 2 above)
Comment (lkocman) [reply gocd/xcdchk.jump.opensuse.yaml]: I did use a separate pipeline

  XCDCHK.openSUSE_Jump:
    group: openSUSE.XCDCHK
    lock_behavior: unlockWhenFinished

Does the group look like a good match?

**Comment by coolo** [comment]
is this supposed to be continoued?

**Comment by coolo** [comment]
I guess not

**Pull request closed without merging**